examples/hpc/dla-driver/src/mmap.rs

```diff
@@ -71,17 +72,17 @@ pub(crate) const DLA_CTRL_VLD_BITMASK_UNUSED: usize = 0b10000000000;
 pub(crate) const DLA_PP_CLIP_BITMASK: usize = 0b111110000000000000000;
 
 pub(crate) const DLA_BUF_INPUT: usize = 0x14;
-pub(crate) const DLA_BUF_INPUT_WIDTH_OFFSET: usize = 0x0;
-pub(crate) const DLA_BUF_INPUT_HEIGHT_OFFSET: usize = 0x9;
-pub(crate) const DLA_BUF_INPUT_CHANNELS_OFFSET: usize = 0x18;
```

Comment:
Double check these changes are correct. We are changing the offset from `0x18` to `18` which means that the value changes from 24 to 18.

Reply:
This is in fact correct. The hexadecimal value was a typo that started causing issues only now.